charts/radar-appserver/templates/ingress.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.upload.enabled }}
```

Comment:
Why not just add it for the annotation configuration instead of creating a new value and condition?

Reply:
You mean that it is always present? I reasoned that when not needed it would be more secure to not add it.

Reply:
Ok, I will remove the conditional inclusion.

Reply:
Thanks!